### PR TITLE
Log cleanups

### PR DESCRIPTION
**Pull request by fifieldt**
change log levels, shorten log messages, delete commented out logs

### CODE REVIEW

**Comment by fifieldt** [comment]
For your consideration, as an example. If this is a good idea will continue to make similar changes.

File: src/gps/GPS.cpp [hunk 1]
```diff
@@ -999,7 +981,7 @@ int32_t GPS::runOnce()
                       GNSS_MODEL_UBLOX10)) {
             // reset the GPS on next bootup
             if (devicestate.did_gps_reset && scheduling.elapsedSearchMs() > 60 * 1000UL && !hasFlow()) {
-                LOG_DEBUG("GPS is not communicating, trying factory reset on next bootup.");
+                LOG_DEBUG("GPS is not communicating, trying factory reset on next boot.");
```
Comment (thebentern) [inline src/gps/GPS.cpp:984]: Should we get rid of periods at the end of sentences as well? :-)

**Review by thebentern — APPROVED** [review]

File: src/mesh/PhoneAPI.cpp [hunk 2]
```diff
@@ -451,7 +448,7 @@ size_t PhoneAPI::getFromRadio(uint8_t *buf)
 
 void PhoneAPI::sendConfigComplete()
 {
-    LOG_INFO("getFromRadio=STATE_SEND_COMPLETE_ID");
+    LOG_INFO("Config Send Complete.");
     fromRadioScratch.which_payload_variant = meshtastic_FromRadio_config_complete_id_tag;
```
Comment (caveman99) [inline src/mesh/PhoneAPI.cpp:452]: I would change all intermediate steps to LOG_DEBUG till the Config Send Complete is issued. Start and stop of config transfer is an info.

File: src/mesh/RadioLibInterface.cpp [hunk 3]
```diff
@@ -167,7 +167,7 @@ ErrorCode RadioLibInterface::send(meshtastic_MeshPacket *p)
         }
 
     } else {
-        LOG_WARN("send - lora tx disabled because RegionCode_Unset");
+        LOG_WARN("send - lora tx disabled: Region is not set");
         packetPool.release(p);
```
Comment (caveman99) [inline src/mesh/RadioLibInterface.cpp:171]: "lora tx disabled: region unset"

File: src/mesh/RadioLibInterface.cpp [hunk 4]
```diff
@@ -383,7 +372,7 @@ void RadioLibInterface::handleReceiveInterrupt()
 
 #ifndef DISABLE_WELCOME_UNSET
     if (config.lora.region == meshtastic_Config_LoRaConfig_RegionCode_UNSET) {
-        LOG_WARN("recv - lora rx disabled because RegionCode_Unset");
+        LOG_WARN("lora rx disabled: Region is not set");
         airTime->logAirtime(RX_ALL_LOG, xmitMsec);
```
Comment (caveman99) [inline src/mesh/RadioLibInterface.cpp:376]: Same as above ...

File: src/modules/PositionModule.cpp [hunk 5]
```diff
@@ -449,23 +449,6 @@ struct SmartPosition PositionModule::getDistanceTraveledSinceLastSend(meshtastic
     float distanceTraveledSinceLastSend = GeoCoord::latLongToMeter(
         lastGpsLatitude * 1e-7, lastGpsLongitude * 1e-7, currentPosition.latitude_i * 1e-7, currentPosition.longitude_i * 1e-7);
 
-#ifdef GPS_EXTRAVERBOSE
-    LOG_DEBUG("--------LAST POSITION------------------------------------");
```
Comment (caveman99) [inline src/modules/PositionModule.cpp:453]: i would change all the GPS_EXTRAVERBOSE stuff to GPS_DEBUG. It's been useful in the past to just enable that.

**Review by caveman99 — CHANGES_REQUESTED** [review]

**Comment by fifieldt** [comment]
Thanks for the reviews guys, that's helpful to understand how you like your logs. I'll make some fixes and upload a new version.

File: src/gps/GPS.cpp (hunk 1 above)
Comment (jp-bennett) [reply src/gps/GPS.cpp:984]: You joke, but that probably represents nearly as much flash as the \n purge.

Comment (thebentern) [reply src/gps/GPS.cpp:984]: I'm actually not joking. I don't need proper punctuation in my logs. Haha

File: src/gps/GPS.cpp [hunk 6]
```diff
@@ -794,6 +794,9 @@ void GPS::writePinEN(bool on)
 
     // Write and log
     enablePin->set(on);
+#ifdef GPS_DEBUG
+    LOG_DEBUG("Pin EN %s", val == HIGH ? "HIGH" : "LOW");
```
Comment (thebentern) [inline src/gps/GPS.cpp:798]: HI instead of HIGH?

**Comment by fifieldt** [comment]
Will pause here -- don't want to add too many changes in one patch; too hard to review :)

File: src/gps/GPS.cpp [hunk 7]
```diff
@@ -1564,13 +1549,13 @@ bool GPS::lookForLocation()
...
     if (toDegInt(loc.lat) > 900000000) {
-#ifdef GPS_EXTRAVERBOSE
+#ifdef GPS_DEBUG
         LOG_DEBUG("Bail out EARLY on LAT %i", toDegInt(loc.lat));
 #endif
         return false;
     }
     if (toDegInt(loc.lng) > 1800000000) {
-#ifdef GPS_EXTRAVERBOSE
+#ifdef GPS_EDEBUG
```
Comment (thebentern) [inline src/gps/GPS.cpp:1558]: EDEBUG? :D

**Review by thebentern — APPROVED** [review]

**Pull request merged**